### PR TITLE
Add blog post for latest standard call

### PR DESCRIPTION
**Pull request by Ainali**
Solves #119

-----
[View rendered _posts/2020-11-16-notes-from-community-call-5-november-2020.md](https://github.com/publiccodenet/blog/blob/November-standard-call/_posts/2020-11-16-notes-from-community-call-5-november-2020.md)

### CODE REVIEW

File: _posts/2020-11-16-notes-from-community-call-5-november-2020.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+title: "Notes from community call - 5 November 2020"
+date: 2020-11-16
+author: Jan Ainali
+type: blogpost
+excerpt: Discussion about challenges in implementing the standard
```
Comment (ElenaFdR) [inline _posts/2020-11-16-notes-from-community-call-5-november-2020.md:6]:
```suggestion
excerpt: Discussion about challenges in implementing the Standard
```

File: _posts/2020-11-16-notes-from-community-call-5-november-2020.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## Attendees
+
+* [Jan Ainali](https://publiccode.net/team/jan-ainali.html)
+* [Alba Roza](https://publiccode.net/team/alba-roza.html)
+* [Charlotte Heikendorf](https://os2.eu/bruger/charlotte-heikendorf), OS2
+
+## Notes
+
+During the call, we discussed different reasons why it sometimes is a challenge implementing the Standard for Public Code. It covered several different aspects relating to governance (Who is making the call to comply with the Standard?), language (Are the decision makers comfortable with adopting a standard only described in English?) and committment (What does it mean for the organisation to commit to comply with the Standard?).
```
Comment (ElenaFdR) [inline _posts/2020-11-16-notes-from-community-call-5-november-2020.md:27]:
```suggestion
During the call, we discussed why implementing the Standard for Public Code can sometimes be challenging. We covered several different aspects relating to governance (i.e., who decides to comply with the Standard?), language (i.e., are the decision makers comfortable with adopting a standard only described in English?) and committment (i.e., what does it mean for the organisation to commit to comply with the Standard?).
```

**Review by ElenaFdR — COMMENTED** [review]
Sentence tweak proposal, otherwise ready to go!

File: _posts/2020-11-23-notes-from-community-call-5-november-2020.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+## Attendees
+
+* [Jan Ainali](https://publiccode.net/team/jan-ainali.html)
+* [Alba Roza](https://publiccode.net/team/alba-roza.html)
+* [Charlotte Heikendorf](https://os2.eu/bruger/charlotte-heikendorf), OS2
+
+## Notes
+
+During the call, we discussed why implementing the Standard for Public Code can sometimes be challenging. We covered several different aspects relating to governance (i.e., who decides to comply with the Standard?), language (i.e., are the decision makers comfortable with adopting a standard only described in English?) and committment (i.e., what does it mean for the organisation to commit to comply with the Standard?).
```
Comment (ElenaFdR) [inline _posts/2020-11-23-notes-from-community-call-5-november-2020.md:27]:
```suggestion
During the call, we discussed why implementing the Standard for Public Code can sometimes be challenging. We covered several different aspects relating to governance (i.e., who decides to comply with the Standard?), language (i.e., are the decision makers comfortable with adopting a standard only described in English?) and commitment (i.e., what does it mean for the organization to commit to comply with the Standard?).
```

**Review by ElenaFdR — APPROVED** [review]

**Review by AlbaRoza — COMMENTED** [review]
Too late for my review, I think. It seems fine anyway!

**Pull request merged**